db/models/taskResult.go

```diff
@@ -61,18 +61,19 @@ type TsIcmpResult struct {
 type TsHopResult struct {
```

Comment:
@arthureld may you also rename the table and model from`TsHopResult` to `TsTracerouteResultHop`(the table name also) Will be a bit more clear which results this table stores. E.g. we have TsDnsResultAnswer and TsDnsResult. Thanks!